paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditCardDetailsInteractor.kt

```diff
@@ -22,14 +25,75 @@ internal typealias CardUpdateParamsCallback = (CardUpdateParams?) -> Unit
 
 internal typealias CardBrandCallback = (CardBrand) -> Unit
 
+internal data class EditCardPayload(
+    val last4: String?,
+    val expiryMonth: Int?,
+    val expiryYear: Int?,
+    val brand: CardBrand,
```

Comment:
just to confirm, the card brand is the only field that's not nullable here?

Reply:
Yep. For the other fields, we're basically just carrying over the nullability from the `PaymentMethod` fields.